database_cleanup/tests/test_database_cleanup.py

```diff
@@ -172,3 +171,4 @@ def tearDown(self):
                     "DELETE FROM ir_model WHERE id in %s", (tuple(self.models.ids),)
                 )
             cr2.commit()
+            super(TestDatabaseCleanup, self).tearDown()
```

Comment:
Trivial, but scoping the call to super under the cursor contextmanager makes less sense. Just unindent.

Reply:
> Trivial, but scoping the call to super under the cursor contextmanager makes less sense. Just unindent.

Stupid me. Will do.